cpp/src/parquet/metadata.h

```diff
@@ -20,13 +20,15 @@
 #include <cstdint>
 #include <map>
 #include <memory>
+#include <optional>
 #include <string>
 #include <utility>
 #include <vector>
 
 #include "parquet/platform.h"
 #include "parquet/properties.h"
 #include "parquet/schema.h"
+#include "parquet/statistics.h"
```

Comment:
We should try to be parcimonious when adding inclusions in header files, as they increase compile times. This one doesn't seem necessary as `class EncodedStatistics` is forward-declared below.

Reply:
Done.

Reply:
You said "done" but didn't address this?

Reply:
Oops, not sure what happened there. It should be removed now.